samples/snippets/create_training_pipeline_image_classification_sample.py

```diff
@@ -30,18 +30,18 @@ def create_training_pipeline_image_classification_sample(
     # Initialize client that will be used to create and send requests.
     # This client only needs to be created once, and can be reused for multiple requests.
     client = aiplatform.gapic.PipelineServiceClient(client_options=client_options)
-    training_task_inputs_dict = {
-        "multiLabel": True,
-        "modelType": "CLOUD",
-        "budgetMilliNodeHours": 8000,
-        "disableEarlyStopping": False,
-    }
-    training_task_inputs = json_format.ParseDict(training_task_inputs_dict, Value())
+
+    icn_training_inputs = definition.AutoMlImageClassificationInputs(
```

Comment:
Note that here we introduce an inconsistent style of using instance of a particular python class, whereas the rest of the sample is in python dicts. I think technically we could keep using python dict here too (with only the change of camelCase to snake_case for dict keys).

 @leahecole let us know if this is fine according to sample style guidelines.  Several more samples will be updated and follow the same pattern as the two samples of this PR.

Reply:
General consensus amongst the owners was preference to have generated classes for API resources - having spell check and autocomplete as well as knowing where to look in reference docs is helpful

If you're using a user-defined object with arbitrary properties, a dict may be simpler.

Reply:
This may be a place where we need to avoid imposing consistency across the board.  For example explicitly construct instances can make certain samples (e.g. https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/create_hyperparameter_tuning_job_python_package_sample.py) much more difficult to read than dicts, and in some cases we are forced to use instances (e.g. https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/upload_model_explain_image_managed_container_sample.py) because of dependency issues.

Reply:
Just to clarify: I'm keeping this sample as-is?